.github/workflows/installer.yml

```diff
@@ -98,7 +98,7 @@ jobs:
       # We cannot use v4, as it requires a newer glibc version than some of the
       # tested images provide. See
       # https://github.com/actions/checkout/issues/1487
-      uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3.6.0
```

Comment:
Will need to see if we can ignore specific files from dependabot, or otherwise disable PRs for actions/checkout, as bumping this would break the installer flow.